src/views/index_view.js

```diff
@@ -113,20 +113,20 @@ export default BaseView.extend({
   },
 
   // Adjust the height of the iframe to match the height of the body.
-  adjustHeight () {
+  adjustHeight() {
     webChannel.sendAdjustHeight(this.el.offsetHeight);
   },
 
   // Clear results of each of our adapters.
-  resetAdapters () {
-    let adapters = [
+  resetAdapters() {
+    const adapters = [
       activityResultsAdapter.results,
       recommendedAdapter.results,
       searchSuggestionsAdapter.localSuggestions,
       searchSuggestionsAdapter.remoteSuggestions,
       searchSuggestionsAdapter.combinedSuggestions
     ];
-    for (let adapter in adapters) {
+    for (const adapter in adapters) { // eslint-disable-line guard-for-in
```

Comment:
Actually, `adapters` is an array. There's no reason to use for-in to iterate it, we can just do `adapters.forEach`.